liubai-frontends/liubai-vscode-extension/src/utils/Logger.ts

```
@@ -0,0 +1,33 @@
+const customEnv = liuEnv.getEnv()
+
+export class Logger {
+  private static outputChannel: vscode.LogOutputChannel
+
+  static init(context: vscode.ExtensionContext) {
+    const appName = customEnv.appName
+    this.outputChannel = vscode.window.createOutputChannel(appName, { log: true })
+  }
+
+  static info(message: string, ...args: any[]) {
+    this.outputChannel.info(message, ...args)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)